app/src/main/java/io/github/droidkaigi/confsched/viewmodel/SessionDetailViewModel.java

```diff
@@ -0,0 +1,123 @@
+package io.github.droidkaigi.confsched.viewmodel;
+
+import android.content.Context;
+import android.databinding.BindingAdapter;
+import android.support.annotation.NonNull;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v4.content.ContextCompat;
+import android.text.TextUtils;
+import android.text.util.Linkify;
+import android.view.View;
+import android.widget.TextView;
+
+import java.util.Date;
+
+import javax.inject.Inject;
+
+import io.github.droidkaigi.confsched.R;
+import io.github.droidkaigi.confsched.dao.SessionDao;
+import io.github.droidkaigi.confsched.model.Session;
+import io.github.droidkaigi.confsched.util.AlarmUtil;
+import io.github.droidkaigi.confsched.util.DataBindingAttributeUtil;
+import io.github.droidkaigi.confsched.util.DateUtil;
+import io.github.droidkaigi.confsched.util.PageNavigator;
+import io.github.droidkaigi.confsched.viewmodel.event.EventBus;
+import io.github.droidkaigi.confsched.viewmodel.event.SessionSelectedChangedEvent;
+
+public class SessionDetailViewModel implements ViewModel {
+
+    private final Context context;
+    private final PageNavigator navigator;
+    private final SessionDao dao;
+    private final EventBus eventBus;
+
+    public Session session;
+
+    @Inject
+    public SessionDetailViewModel(Context context,
+                                  PageNavigator navigator,
+                                  SessionDao dao,
+                                  EventBus eventBus) {
+        this.context = context;
+        this.navigator = navigator;
+        this.dao = dao;
+        this.eventBus = eventBus;
+    }
+
+    @Override
+    public void destroy() {
+        // Do nothing
```

Comment:
Not sure but shouldn't we need to release context, navigator, dao, eventBus? I just wonder context causes memory leaks 🙇